changelog/unreleased/2970

```diff
@@ -0,0 +1,9 @@
+Enhancement: optimize fileChanged check
```

Comment:
I'm afraid that's a bit too technical for most users to understand. The changelog is intended for restic users, who are not familiar with restic internals. What about `Enhancement: Optimize check for unchanged files during backup`?